### PR TITLE
B2

### PR DESCRIPTION
**Pull request by AnithaPriya1386**
index.html and style.css is updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <div class="container">
-      <h1 class="heading">English Dictionary</h1>
+      <h1 class="heading">Beethovan English Dictionary </h1>
```
Comment (AnithaPriya1386) [inline index.html]:
```suggestion
      <h1 class="heading">Oxford English Dictionary </h1>

```

File: style.css [hunk 2]
```diff
@@ -4,7 +4,7 @@ body{
     min-height: 100vh;
     justify-content: center;
     align-items: center;
-    background-color: salmon;
+    background-color: rgb(213, 167, 58);
```
Comment (AnithaPriya1386) [inline style.css]:
```suggestion
    background-color: rgb(100, 167, 58);

```

File: index.html (hunk 1 above)
Comment (Anithapriya86) [inline index.html]:
```suggestion
      <h1 class="heading">Oxford English Dictionary </h1>

```

File: style.css (hunk 2 above)
Comment (Anithapriya86) [inline style.css]:
```suggestion
    background-color: rgb(100, 167, 58);

```

**Review by Anithapriya86 — CHANGES_REQUESTED** [review]
Files have been edited to make some changes

**Review by AnithaPriya1386 — COMMENTED** [review]
Ok Done, commited successfully

**Pull request merged**